tutorials/sample_vqe_program/qiskit_runtime_vqe_program.ipynb

```diff
@@ -0,0 +1,1394 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(interm_results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "59f6b692",
+   "metadata": {},
+   "source": [
+    "### Formatting the returned results\n",
+    "\n",
+    "In order to format the return results into the desired format, we need to specify a decoder.  This decoder must have a `decode` method that gets called to do the actual conversion.  In our case `OptimizeResult` is a simple sub-class of `dict` so the formatting is simple."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "id": "262f9d38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.providers.ibmq.runtime import ResultDecoder\n",
+    "from scipy.optimize import OptimizeResult\n",
+    "\n",
+    "class VQEResultDecoder(ResultDecoder):\n",
+    "    @classmethod\n",
+    "    def decode(cls, data):\n",
+    "        data = super().decode(data)  # This is required to preformat the data returned.\n",
+    "        return OptimizeResult(data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a8681b5",
+   "metadata": {},
+   "source": [
+    "We can then use this when returning the job result:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "id": "afb35c33",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "     fun: -1.466552734375\n",
+       " message: 'Optimization terminated successfully.'\n",
+       "    nfev: 20\n",
+       "     nit: 10\n",
+       " success: True\n",
+       "       x: array([ 1.14529875,  3.59359094,  1.52406932,  4.29224294,  6.1804034 ,\n",
+       "        4.49658598,  5.48578434,  2.35112148,  6.43273259,  4.21766161,\n",
+       "        3.56638848,  1.52288587,  1.63994411,  2.3065571 ,  0.84399359,\n",
+       "       -0.03350545])"
+      ]
+     },
+     "execution_count": 27,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "job3 = provider.runtime.run(program_id, options=options, inputs=inputs)\n",
+    "job3.result(decoder=VQEResultDecoder)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ade26bc9",
+   "metadata": {},
+   "source": [
+    "## Simplifying program execution with wrapping functions\n",
+    "\n",
+    "While runtime programs are pwoerful and flexible, they are not the most friendly things to interact with.  Therefore if your program is intended to be used by others it is best to make wrapper functions and/or classes that simply the user experience.  Moreoever, such wrappers allow for validation of user inputs client side, which can quickly find errors that would otherwise be raised latter during the execution process; something that might have taken hours waiting in queue to get to.\n",
```

Comment:
```suggestion
    "While runtime programs are powerful and flexible, they are not the most friendly things to interact with.  Therefore if your program is intended to be used by others it is best to make wrapper functions and/or classes that simplify the user experience.  Moreover, such wrappers allow for validation of user inputs on the client side, which can quickly find errors that would otherwise be raised latter during the execution process; something that might have taken hours waiting in queue to get to.\n",
```

Reply:
I think Abby fixed all of these in the later version